subiquity/server/controllers/locale.py

```diff
@@ -66,4 +67,9 @@ async def GET(self) -> str:
 
     async def POST(self, data: str):
         self.model.switch_language(data)
+        # In an answers based install, the client will POST "" as the
+        # locale/data if there is no locale section. We should probably use
+        # null instead but ...
+        if not self.model.selected_language:
```

Comment:
I don't understand this check. But I got up at 04:30 so it might be my issue.

Reply:
answers can /currently/ pass an empty string, so the check is here to avoid calling try_gen_localedef when `self.model.selected_language == ""`.

Otherwise, the `lang, charset = self.model.selected_language.split(".")` would raise an exception.

Reply:
Well, it does not make sense. I guess the condition is inverted. I must have tried before refactoring :facepalm:

Reply:
ah ok :)